src/server/config.js

```diff
@@ -1,16 +1,7 @@
 require('dotenv').config();
-const debug = require('./debug');
+const globalErrorHandlers = require('./globerrorhandlers');
```

Comment:
I guess it's more correct to do it in the app's entry point script, don't you think so?